functions/http/index.js

```diff
@@ -0,0 +1,34 @@
+const http = async ({
+  url,
+  method,
+  body,
+  headers,
+  protocol,
+  queryParameters,
+}) => {
+  const queryString = queryParameters
```

Comment:
Maybe we could move this to a separate function, something like `parseQueryParameters`